### PR TITLE
bugfix/15896-gantt-scrollbar

### LINKED ISSUE #15896: Gantt chart scroll bar issue on upgrade to 9.1.1
#### Expected behaviour
Gantt Chart should show Y axis scroll bar with below settings

		yAxis:{
					max:2,
            scrollbar: {
                enabled: true,
            },
			},

This works fine with the version 8.0.0, when upgraded to 9.1.1, scroll bar disappeared with the same config

In the provided jsfiddle, which is pointing to 8.0.0, you will see the scrollbar, but if you change it to point to 9.1.1, the scrollbar will disappear. Kindly guide us what needs to be done for scroll bar to appear in version 9.1.1

#### Live demo with steps to reproduce
https://jsfiddle.net/sreejithga/ph703yz8/6/

#### Product version
9.1.1

#### Affected browser(s)
Chrome

### PR DESCRIPTION
**Pull request by khlieng**
Fixed #15896, scrollbar did not work with gantt.

### CODE REVIEW

File: ts/masters/modules/stock.src.ts [hunk 1]
```diff
@@ -22,8 +22,10 @@ import '../../Core/Navigator.js';
...
 const G: AnyRecord = Highcharts;
+// Compositions
+if (!G.Scrollbar) {
+    Scrollbar.compose(G.Axis);
+}
 // Classes
 G.Scrollbar = Scrollbar;
 G.StockChart = G.stockChart = StockChart.stockChart;
-// Compositions
-Scrollbar.compose(G.Axis);
```
Comment (bre1470) [inline ts/masters/modules/stock.src.ts:29]: The order should not be important for the composition here. (= compositions usually come last, after all namespace assignments) Was something failing?

**Review by bre1470 — APPROVED** [review]

**Review by bre1470 — CHANGES_REQUESTED** [review]
Wait a second. The change means, that composition has no detection agains running twice. This should be fixed with a private array in the compose function, taking all modified axis class and then next time check, if the array already contains the axis class to modify.

**Comment by khlieng** [comment]
> Wait a second. The change means, that the composition has no detection agains running twice. This should be fixed with a private array in the compose function, taking all modified axis class and then next time check, if the array already contains the axis class to modify.

Doesnt the `if (!G.Scrollbar)` handle that?

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Pending bre1420's comment

**Comment by bre1470** [comment]
Without fixing the composition only the master is compatible to tree shaking. If a user calls the composition in a ES6 module it has not protection agains double modification. (which triggers the bug)

**Review by bre1470 — APPROVED** [review]
Thank you! Looks good. 👍

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**